packages/core/echo/feed/src/sync.test.ts

```
@@ -3,101 +3,370 @@
 //
 
 import type * as SqlClient from '@effect/sql/SqlClient';
-import * as SqliteClient from '@effect/sql-sqlite-node/SqliteClient';
-import { describe, expect, it } from '@effect/vitest';
+import { describe, expect, it, test } from '@effect/vitest';
 import * as Effect from 'effect/Effect';
 import * as ManagedRuntime from 'effect/ManagedRuntime';
+import * as SqlExport from '@dxos/sql-sqlite/SqlExport';
+import { SqlTransaction } from '@dxos/sql-sqlite';
+import { layerMemory } from '@dxos/sql-sqlite/platform';
+import { Array } from 'effect';
```

Biome (2.3.13)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)